components/tabs/tabs.js

```diff
@@ -153,7 +150,6 @@ class Tabs extends Component {
 
 Tabs.defaultProps = {
   activeTab: '0',
-  open: false,
```

Comment:
I've noticed that this prop is not used in the component, thus, I'm deleting it. Please correct me if I am wrong